content/code-style.md

```diff
@@ -0,0 +1,172 @@
+
+```js
+// This code is misleading because it looks like both statements
+// are inside the conditional.
+if (condition)
+  firstStatement();
+  secondStatement();
+```
+
+```js
+// Much clearer!
+if (condition) {
+  firstStatement();
+}
+
+secondStatement();
+```
+
+<h3 id="standard-tools">Use standard tools</h3>
+
+Having a consistent style means that it's easier to adopt standard tools for error checking. For example, if you adopt a convention that you must always use `let` or `const` instead of `var`, you can now use a tool to ensure all of your variables are block-scoped. That means one less thing to thing about when reading code.
+
+If your whole organization uses the same tool to check code, you can more easily read code written by anyone else. If we can standardize across the Meteor community, this will also be the case for open source projects and tutorials! In this article, we'll give suggestions for standard linter and transpiler configurations for Meteor.
+
+<h3 id="code-samples">Use code samples directly</h3>
+
+If you're using the standard Meteor style guide, you can more easily understand and copy code samples from the Meteor Guide, documentation, tutorials, example apps, and more, without having to reformat them first.
+
+For the reasons above and more, we believe that the overall benefits of having a consistent style outweigh opinions on the individual decisions. For this reason, we've adopted a very popular JavaScript style guide from AirBnB basically as-is. We think the Meteor version is a bit clearer and removes some unnecessary content, but most things are the same.
+
+<h2 id="javascript">JavaScript style</h2>
+
+Here at Meteor, we strongly believe that JavaScript is the best language to build web applications, for a variety of reasons. JavaScript is constantly improving, and the standards around ES2015 have really brought together the JavaScript community. Here are our recommendations about how to use ES2015 JavaScript in your app today.
+
+[XXX Ben's .gif of ecmascript here]
+
+<h3 id="ecmascript">Use the `ecmascript` package</h3>
+
+Meteor comes with a core package called `ecmascript` that transpiles all `.js` files in your app to support a stable list of modern JavaScript standards that are reliable for production use. This means that not all cutting-edge features, like function decorators or generators, are supported; we think it's best to only build your app on standards that are good bets for future browser support.
```

Comment:
The real answer is this file: https://github.com/meteor/meteor/blob/devel/packages/ecmascript/README.md